contrib/gofiber/fiber.v2/fiber.go

```diff
@@ -43,7 +43,7 @@ func Middleware(opts ...Option) func(c *fiber.Ctx) error {
 		opts = append(opts, cfg.spanOpts...)
 		span, _ := tracer.StartSpanFromContext(c.Context(), "http.request", opts...)
 
-		fmt.Printf("Starting Span")
+		fmt.Printf("Starting Span\n")
```

Comment:
This is needed to make gotestsum happy. Otherwise we get [failures parsing the test output](https://gist.github.com/felixge/b152b08956698a5f0ea4be270d0ddc0c).

Reply:
We should probably just remove this line.

Reply:
DONE. 4732d343307002cf2e5c53915c740be4d1653131